graphql/resolvers.go

```diff
@@ -782,10 +782,7 @@ func (r *patchResolver) TaskCount(ctx context.Context, obj *restModel.APIPatch)
 
 func (r *patchResolver) BaseVersionID(ctx context.Context, obj *restModel.APIPatch) (*string, error) {
 	baseVersion, err := model.VersionFindOne(model.BaseVersionByProjectIdAndRevision(*obj.ProjectId, *obj.Githash).Project(bson.M{model.VersionIdentifierKey: 1}))
-	if err != nil {
-		return nil, InternalServerError.Send(ctx, fmt.Sprintf("Error getting base version ID for patch %s: %s", *obj.Id, err.Error()))
-	}
-	if baseVersion == nil {
+	if baseVersion == nil || err != nil {
```

Comment:
Looks like the query already returns nil for the error if there are no results. So it looks like if theres an error it might actually be a real error do we still want to ignore it? 
https://github.com/evergreen-ci/evergreen/blob/bdba81973a413c98df9d289fe8c6975847787767/model/version_db.go#L249-L253